src/partials/carousel.hbs

```diff
@@ -0,0 +1,28 @@
+<div class="banner">
+  <div id="carousel" class="carousel carousel-container">
+    <div class="slides">
+      <div  class='item slide' data-state="active">
+        <img src='static/images/offers/offer1.jpg' alt='1' />
+      </div>
+      <div  class='item slide'>
+        <img src='static/images/offers/offer2.jpg' alt='2' />
+      </div>
+      <div  class='item slide'>
+        <img src='static/images/offers/offer3.jpg' alt='3' />
+      </div>
+      <div  class='item slide'>
+        <img src='static/images/offers/offer4.jpg' alt='4' />
+      </div>
+      <div  class='item slide'>
+        <img src='static/images/offers/offer5.jpg' alt='5' />
+      </div>
+    </div>
+    <div class="indicators">
+      <input class="indicator"  ="indicator" data-slide="1" data-state="active" checked type="radio" />
```

Comment:
I hope you will make it dynamic based on JSON contract

Reply:
not doing it right now it'l take a lot of bandwidth